README.md

```diff
@@ -214,6 +214,38 @@ The second solution is to add a new alias to your mix config that recompiles ass
 
 This method is less error prone but it will cause a delay when starting your test suite.
 
+#### LiveView
+
+In order to test Phoenix LiveView (as of [version 0.17.7](https://github.com/phoenixframework/phoenix_live_view/releases/tag/v0.17.7)) with Wallaby you'll also need to add a function to each `mount/3` function in your LiveViews, or use the `on_mount` `live_session` lifecycle hook in the router:
+
+```elixir
+defmodule MyApp.Hooks.AllowEctoSandbox do
+  import Phoenix.LiveView
+
+  def on_mount(:default, _params, _session, socket) do
+    allow_ecto_sandbox(socket)
+    {:cont, socket}
+  end
+
+  defp allow_ecto_sandbox(socket) do
+    %{assigns: %{phoenix_ecto_sandbox: metadata}} =
+      assign_new(socket, :phoenix_ecto_sandbox, fn ->
+        if connected?(socket), do: get_connect_info(socket, :user_agent)
+      end)
+
+    Phoenix.Ecto.SQL.Sandbox.allow(metadata, Ecto.Adapters.SQL.Sandbox)
```

Comment:
Nice one for doing this! I had to set it up last week and having this explicit documentation will really help.

One thing to add is that we currently recommend using a config key to define the sandbox:

``` elixir
config :your_app, :sandbox, Ecto.Adapters.SQL.Sandbox
```

So then this line could be:

``` elixir
Phoenix.Ecto.SQL.Sandbox.allow(metadata, Application.get_env(:your_app, :sandbox))
```

That then makes it continue to work properly if/when you define a custom sandbox module to be able to sandbox other things than the database connection (e.g. Mox mocks -- see the current Wallaby docs).